### PR TITLE
Helmchart deployment

### PR DESCRIPTION
**Pull request by dbw7**
This is the complete helm-chart air-gapped deployment. It is only missing tests and separate log files for different helm commands.

Made a new PR cause I shredded the old branch by accident.

### CODE REVIEW

File: pkg/registry/helm.go [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+	if localHelmSrcDir == "" {
+		return nil, nil
+	}
+
+	helmManifestPaths, err := getLocalManifestPaths(localHelmSrcDir)
+	if err != nil {
+		return nil, fmt.Errorf("getting helm manifest paths: %w", err)
+	}
+
+	var allManifests [][]map[string]any
```
Comment (atanasdinov) [inline pkg/registry/helm.go:281]: One slice should be more than enough. Any reasons for the nesting?

File: Dockerfile [hunk 2]
```diff
@@ -24,7 +24,8 @@ RUN zypper install -y \
     xorriso squashfs  \
     libguestfs kernel-default e2fsprogs parted gptfdisk btrfsprogs \
     podman \
-    createrepo_c
+    createrepo_c \
+    helm
```
[the remark this answers is not in the document]
Comment (atanasdinov) [reply Dockerfile:28]: Sorry, this is being used in the chart manipulation logic. Please ignore.

File: pkg/combustion/registry.go [hunk 3]
```diff
@@ -268,3 +339,108 @@ func writeRegistryMirrors(ctx *image.Context, hostnames []string) error {
...
+	}
+
+	for i, manifest := range manifests {
+		for j, doc := range manifest {
+			data, err := yaml.Marshal(doc)
+			if err != nil {
+				return fmt.Errorf("marshaling data: %w", err)
+			}
+
+			fileName := fmt.Sprintf("manifest%d.yaml", i+j)
```
[the remark this answers is not in the document]
Comment (dbw7) [reply pkg/combustion/registry.go:423]: Resolved by fix to another comment

**Comment by dbw7** [comment]
Closing in favor of #178 and #179 .

**Pull request closed without merging**